src/views/templates/index.ts

```diff
@@ -3,4 +3,5 @@ import TemplatesView from './TemplatesView';
 export * from './search-categories';
 export * from './template-search';
 export * from './template-tiles';
+export * from './template-details';
```

Comment:
Template-details is separate view so should be located directly after views/template-details and should have structure like templates-view. 
So structure should look like:
views
   templates
   login
   template-details - here put your code

Reply:
Done